static/installer.sh

```diff
@@ -2,12 +2,14 @@
 
 set -uxo pipefail
 
-tarantool_default_version="2.8"
+tarantool_default_version="2"
 repo_type="live"
 repo_path=""
 repo_only=0
 os=""
 dist=""
+arch=""
```

Comment:
It seems it's not needed.

Reply:
Yes, as it is used as a local variable now

Reply:
removed